src/enums/ErrorCode.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+
+final class ErrorCode extends BaseEnum

```

Comment:
Instead of actual Enum's, let's simplify by doing something like this: 
`define("CONFIG_NOT_FOUND", "The config.json file wasn't found.");`
Then we can get rid of the BaseEnum and translation files.